packages/auth-sveltekit/readme.md

```diff
@@ -0,0 +1,161 @@
+It supports authentication with various OAuth providers, as well as email/password authentication.
+
+## Installation
+
+This package is published on npm and can be installed with your package manager of choice.
+
+```bash
+npm install @edgedb/auth-sveltekit
+```
+
+## Setup
+
+**Prerequisites**: Before adding EdgeDB auth to your Sveltekit app, you will first need to enable the `auth` extension in your EdgeDB schema, and have configured the extension with some providers (you can do this in CLI or EdgeDB UI). Refer to the auth extension docs for details on how to do this.
+
+1. Initialize the client auth helper by passing configuration options to `createClientAuth()`. This will return a `ClientAuth` object which you can use in your components. You can skip this part if you find it unnecessary and provide all your data through the load functions (the next step), but we suggest having the client auth too and use it directly in your components to get OAuth, BuiltinUI and signout URLs.
+
+```ts
+// src/lib/auth.ts
+
+import createClientAuth, {
+  type AuthOptions,
+} from "@edgedb/auth-sveltekit/client";
+
+export const options: AuthOptions = {
+  baseUrl: "http://localhost:5173",
+  // ...
+};
+
+const auth = createClientAuth(options);
+
+export default auth;
+```
+
+The available auth config options are as follows:
+
+- `baseUrl: string`, _required_, The url of your application; needed for various auth flows (eg. OAuth) to redirect back to.
+- `authRoutesPath?: string`, The path to the auth route handlers, defaults to `'auth'`, see below for more details.
+- `authCookieName?: string`, The name of the cookie where the auth token will be stored, defaults to `'edgedb-session'`.
+- `pkceVerifierCookieName?: string`: The name of the cookie where the verifier for the PKCE flow will be stored, defaults to `'edgedb-pkce-verifier'`
+- `passwordResetUrl?: string`: The url of the the password reset page; needed if you want to enable password reset emails in your app.
+
+2. Lets create an EdgeDB client that you will need for creating server auth client.
+
+```ts
+// src/lib/server/auth.ts
+import { createClient } from "edgedb";
+
+export const client = createClient({
+  //Note: when developing locally you will need to set tls  security to insecure, because the dev server uses  self-signed certificates which will cause api calls with the fetch api to fail.
+  tlsSecurity: "insecure",
+});
+```
+
+3. Create the server auth client in a handle hook. Firstly call `serverAuth` passing to it EdgeDB client you created in the previous step, along with configuration options from step 1. This will give you back the `createServerRequestAuth` and `createAuthRouteHook`. You should call `createServerRequestAuth` inside a handle to attach the server client to `event.locals`. Calling `createAuthRouteHook` will give you back a hook so you should just invoke it inside `sequence` and pass your auth route handlers to it.
```

Comment:
This step also needs the user to add a `.d.ts` file with the type of `locals.auth` (if they're using typescript). https://github.com/lucia-auth/examples/blob/main/sveltekit/username-and-password/src/app.d.ts#L5